### DISCUSSION TITLE
fips.dll / fips.so are missing in openssl

### DISCUSSION THREAD

**Discussion by guenterbe** [microsoft/vcpkg#30676] [Q&A] [answered]
If I build openssl from the original sources (obtained from openssl.org), I also get FIPS support in the form of fips.so on Linux and fips.dll on Windows.
After building openssl via vcpkg the FIPS stuff is missing.
How can I control that the fips.so/fips.dll modules are also built?

Thank you,

**Comment by lbermes** [comment, marked as answer]
see [openssl] support fips build feature #30916
